scripts/examples/03-Machine-Learning/00-TensorFlow/tf_image_classification.py

```diff
@@ -43,7 +43,7 @@
     # This combines the labels and confidence values into a list of tuples
     # and then sorts that list by the confidence values.
     sorted_list = sorted(
-        zip(labels, model.predict([img])[0]), key=lambda x: x[1], reverse=True
+        zip(labels, model.predict([img])[0].flatten().tolist()), key=lambda x: x[1], reverse=True
```

Comment:
```suggestion
        zip(labels, model.predict([img])[0].flatten().tolist()), key=lambda x: x[1], reverse=True
```
Does it need to be flattened ? I ask because it was just `tolist()` before, so which one is correct ?

Reply:
Yeah, you have to flatten it because tolist() will generate a list for each dimension. E.g. you'll get things like `[[0.5, 1.0, 3.0, etc.]]`